### DISCUSSION TITLE
how to generate file in original folder structure?

### DISCUSSION THREAD

**Discussion by leamandeep** [egoist/tsup#918] [Q&A]
This is my original folder structure (`src`) &nbsp;  &nbsp;  &nbsp;   and below one is my generated file (`dist`)
![image](https://github.com/egoist/tsup/assets/68345524/94eee1ed-e000-4c12-bc9f-c271d64353b2)  &nbsp;  &nbsp;  &nbsp;   ![image](https://github.com/egoist/tsup/assets/68345524/865147a9-ccb0-4259-8d86-506d7a9fd970)

I want to generate output like the folder structure, I have in the original folder photo.  Also, i want to since it's my custom component library how to optimize the build? And how to generate separate for files?  I guess i have too many questions 😂

**Comment by rj-au** [comment]
This might be related: https://github.com/egoist/tsup/issues/728
